FlowCrypt/Functionality/Services/Client Configuration Service/ClientConfigurationService.swift

```diff
@@ -34,7 +34,8 @@ extension ClientConfigurationService: ClientConfigurationServiceType {
     func fetch(for user: User) async throws -> ClientConfiguration {
         do {
             let raw = try await server.getClientConfiguration(for: user.email)
-            try local.save(for: user, raw: raw)
+            let fesUrl = try await server.getActiveFesUrl(for: user.email)
+            try local.save(for: user, raw: raw, fesUrl: fesUrl)
             return ClientConfiguration(raw: raw)
```

Comment:
from my memory, `getClientConfiguration` also calls `getActiveFesUrl` separately, so we are calling twice. Can we optimize that? 

`getActiveFesUrl` is to figure out if FES is running there or not (once discovered at least once as running, we should save that and always expect it going forward)

`getClientConfiguration` is to fetch the actual config, but it should have to call `getActiveFesUrl` each time if we already saved it as existing at least once.

Reply:
Yes, it'll be fixed by https://github.com/FlowCrypt/flowcrypt-ios/issues/1348 - `fesUrl` will be fetched only once and then reused.

Reply:
ok